Added steps for signing with Azure Code Sign to ELAM driver resource …

jgeurten opened the pull request:
…file info. Need to wait for the PCA cert to be included in pkiops

jgeurten committed:
Added steps for signing with Azure Code Sign to ELAM driver resource file info. Need to wait for the PCA cert to be included in pkiops

PRMerger19 commented:
@jgeurten : Thanks for your contribution! The author(s) have been notified to review your proposed change.

mcleanbyron committed:
Edited new section

PRMerger7 commented:
@mcleanbyron : Thanks for your contribution! The author(s) have been notified to review your proposed change.

jgeurten committed:
Update protecting-anti-malware-services-.md

Added blurb about contacting MVI for questions about onboarding to Azure Code Sign.

PRMerger15 commented:
@jgeurten : Thanks for your contribution! The author(s) have been notified to review your proposed change.

GrantMeStrength commented:
Author waiting for more information.

GrantMeStrength reviewed (changes requested):
I am sorry this PR was not approved when it was submitted.
Is it still accurate?

jgeurten commented:
@GrantMeStrength it is still accurate/required to be documented. The product "Azure Code Sign" has been rebranded to "Trusted Signing"

GrantMeStrength committed:
Update protecting-anti-malware-services-.md

Replacing Azure Code Sign with Trusted Signing

GrantMeStrength committed:
Update protecting-anti-malware-services-.md

GrantMeStrength committed:
Merge branch 'docs' into docs

GrantMeStrength commented:
Updated references to "Azure Code Sign"  to "Trusted Signing"

GrantMeStrength commented:
Is this PR ready to go live?
